pos_product_label/static/src/PrintLabelPopup.xml

```diff
@@ -35,6 +35,7 @@
                                         t-model="data.format"
                                         t-att-id="format"
                                         t-att-value="format"
+                                        t-on-change="updateFormat"
```

Comment:
are you sure this is needed? I thought it'd be handled by the widget through the `t-model="data.format"` attribute